javascript-sdk/examples/run-script-in-headless-browser/run-script-in-headless-browser.js

```diff
@@ -0,0 +1,43 @@
+/* This example demonstrates how to run the script in a headless browser. */
+
+const { wrap } = require('agentql');
+const { chromium } = require('playwright');
+
+// Define the URL of the page to scrape.
+const URL = 'https://scrapeme.live/shop/';
+
+// Define the queries to locate the search box and fetch the stock number.
+const SEARCH_QUERY = `
+{
+    search_products_box
+}
+`;
+
+const STOCK_NUMBER_QUERY = `
+{
+    number_in_stock
+}
+`;
+
+(async () => {
+  // Launch a headless browser using Playwright.
+  const browser = await chromium.launch({ headless: true });
+  const normalPage = await browser.newPage();
+  await normalPage.goto(URL);
+
+  // Wrap a new page instance to use with AgentQL API
+  const wrappedPage = wrap(normalPage);
```

Comment:
Another nit: could we inline page creation? And only call page.goto() after wrap()

Reply:
Hey @frankfeng98, when trying to make this a one liner, I get the following error:

```bash
% node --env-file=.env test.js
node:internal/process/promises:394
    triggerUncaughtException(err, true /* fromPromise */);
    ^

page.evaluate: Execution context was destroyed, most likely because of a navigation
    at PageActivityMonitor.<anonymous> (/Users/zach/CS467/node_modules/agentql/dist/ext/playwright/page-monitor.js:42:29)
    at fulfilled (/Users/zach/CS467/node_modules/agentql/dist/ext/playwright/page-monitor.js:5:58) {
  name: 'Error'
}

Node.js v23.1.0
```

Code snippet:
```js
    // Launch a headless browser using Playwright.
    const browser = await chromium.launch({ headless: false });

    // Create a new page in the browser and wrap it to get access to the AgentQL's querying API
    const page = wrap(await browser.newPage());
    await page.goto(URL);
```

This is why I extracted it to two lines. If this issue only pertains to my node setup then I can change the code back into a one liner but wasn't sure if this affected anyone else. Could you maybe try running this code?

Reply:
Hi Zach! Thank you for flagging this. It help me find a bug in JS SDK. I just updated and published a new version of JS SDK. If you run `npm install`, it should contains the fix. 

Note that in the latest version, `wrap` becomes async, so you need to await it.

Reply:
Thank you for the prompt fix!